packages/server/graphql/types/AzureDevOpsWorkItem.ts

```diff
@@ -14,15 +17,39 @@ const AzureDevOpsWorkItem = new GraphQLObjectType<any, GQLContext>({
   fields: () => ({
     id: {
       type: new GraphQLNonNull(GraphQLID),
-      description: 'GUID instanceId:issueKey'
+      description: 'GUID instanceId:projectKey:issueKey',
+      resolve: ({id, teamProject, url}: {id: string; teamProject: string; url: string}) => {
+        const instanceId = getInstanceId(url)
+        return AzureDevOpsIssueId.join(instanceId, teamProject, id)
+      }
     },
     title: {
       type: new GraphQLNonNull(GraphQLString),
       description: 'Title of the work item'
     },
+    // TODO: change teamProject name: https://github.com/ParabolInc/parabol/issues/7073
     teamProject: {
       type: new GraphQLNonNull(GraphQLString),
-      description: 'Name of the Team Project the work item belongs to'
+      description: 'Name or id of the Team Project the work item belongs to'
+    },
+    project: {
```

Comment:
It feels expensive grabbing the `project` just for the `projectName`, but we can't guarantee the project name from `teamProject` as it's often an `id`. Having access to the `project` in the `workItem` could well be useful in the future, and we currently do this in Jira, so I think it's the right approach

Reply:
agreed!